.travis.yml

```diff
@@ -1,5 +1,4 @@
 rvm:
-  - 1.9.3
```

Comment:
Indeed. Corrected question still stands though: should we support EOL'd software? Users can always continue to use Rails 4, and Ruby 1.9, and paperclip <= 4.2.1.